.github/workflows/check.yml

```diff
@@ -0,0 +1,17 @@
+name: Format check
+
+on: [pull_request]
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
```

Comment:
i think we only want to run this if package-lock.json has been changed, and not on all PR's

Reply:
sure good idea.